doc/api/dns.markdown

```diff
@@ -49,19 +46,20 @@ resolves the IP addresses which are returned.
       });
     });
 
-There are subtle consequences in choosing one or another, please consult the
-[Implementation considerations section][] for more information.
+There are subtle consequences in choosing one over the other another,
+please consult the [Implementation considerations section][] for more
+information.
 
 ## dns.getServers()
 
-Returns an array of IP addresses as strings that are currently being used for
-resolution
+Returns the array of string-encoded IP addresses that are currently being used
+for name resolution.
```

Comment:
I wouldn't consider strings as an encoding, maybe `IP address strings`? Also, not sure about `the`, `an` seems fine to me.